code/espurna/config/sensors.h

```diff
@@ -607,6 +607,25 @@
 #define V9261F_POWER_FACTOR             153699.0
 #define V9261F_RPOWER_FACTOR            V9261F_CURRENT_FACTOR
 
+//------------------------------------------------------------------------------
+// VEML6075 based power sensor
+// Enable support by passing VEML6075_SUPPORT=1 build flag
+//------------------------------------------------------------------------------
+
+#ifndef VEML6075_SUPPORT
+#define VEML6075_SUPPORT                  0
+#endif
+
+#ifndef VEML6075_INTEGRATION_TIME
+#define VEML6075_INTEGRATION_TIME         VEML6075::IT_100MS        // The time, in milliseconds, allocated for a single
+#endif                                                              // measurement. A longer timing budget allows for more
+                                                                    // accurate results at the cost of power.
+
+#ifndef VEML6075_DYNAMIC_MODE
+#define VEML6075_DYNAMIC_MODE             VEML6075::DYNAMIC_NORMAL  // The dynamic mode can either be normal or high. In high
```

Comment:
@xoseperez can these options be configured via the web ui?

Reply:
Yes, but for now it's OK. I will run a migration process on all sensors sometime in the future.